src/crc32c.c

```diff
@@ -0,0 +1,590 @@
+/* crc32c.c -- compute CRC-32C using software table or available hardware instructions
```

Comment:
adjust the path to this file in the exceptions list in contrib/add_license_to_files.jl